calisphere/views.py

```diff
@@ -8,7 +8,7 @@
 from .item_manager import ItemManager
 from .record import Record
 
-from .search_form import (SearchForm, ESSearchForm, solr_escape, 
```

Comment:
Whoops! Here too! `from .utils import query_string_escape`

Reply:
D'oh! I did a global search this time to make sure I wasn't missing some place else!!